DripDotNetTests/SubscriberTests.cs

```diff
@@ -110,7 +107,7 @@ public async Task CanCreateUpdateAndRetrieveSubscriberAsync()
         }
 
         [Fact]
-        public void NonExistantSubcriberReturnsError()
+        public void NonExistedSubscriberReturnsError()
```

Comment:
Thanks for catching a spelling error!  I think we will want to change it slightly, like this:
```cs
public void NonExistentSubcriberReturnsError()
```